Mathlib/Algebra/Group/Subgroup/Defs.lean

```diff
@@ -661,21 +661,36 @@ def setNormalizer (S : Set G) : Subgroup G where
     rw [ha (a⁻¹ * n * a⁻¹⁻¹)]
     simp only [inv_inv, mul_assoc, mul_inv_cancel_left, mul_inv_cancel, mul_one]
 
-variable {H}
+variable {H} {S}
```

Comment:
The variable `S` isn't defined earlier in the file so I think we should specify the type here to help with readability (not sure if there are style conventions for this!)